frontend/src/Editor/Components/components.js

```diff
@@ -365,6 +365,56 @@ export const componentTypes = [
       }
     }
   },
+  {
+    name: 'Radio-button',
+    displayName: 'Radio Button',
+    description: 'A single radio button',
+    component: 'RadioButton',
+    defaultSize: {
+      width: 300,
+      height: 25
+    },
+    others: {
+      showOnDesktop: { type: 'toggle', displayName: 'Show on desktop? '},
+      showOnMobile: { type: 'toggle', displayName: 'Show on mobile?'},
+    },
+    properties: {
+      label: { type: 'code', displayName: 'Label' },
+      value: { type: 'code', displayName: 'Default value' },
+      values: { type: 'code', displayName: 'Option values' },
+      display_values: { type: 'code', displayName: 'Option labels' }
+    },
+    events: {
+      onCheck: { displayName: 'On select'},
+   },
+    styles: {
+      textColor: { type: 'color', displayName: 'Text Color' }
+    },
+    exposedVariables: {},
+    definition: {
+      others: {
+        showOnDesktop: { value: true },
+        showOnMobile: { value : false }
+      },
+      properties: {
+        label: { value: 'Select' },
+        value: { value: '' },
+        values: { value: '{{[1,2,3]}}' },
+        display_values: { value: '{{["one", "two", "three"]}}' },
```

Comment:
Does `yes` and `no` sounds more related to radio buttons?

Reply:
`yes` and `no` with `true` and `false` as values.

Reply:
yeah or `on` or `off`?

Reply:
for sample values, I think `yes` or `no` will be sound better